07_Bootstrap/index.html

```diff
@@ -0,0 +1,198 @@
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse justify-content-end" id="navbarNavDropdown">
+                    <ul class="navbar-nav">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="#">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Features</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Screenshots</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Team</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Pricing</a>
+                        </li>
+                        <li class="nav-item dropdown">
+                            <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
+                                aria-expanded="false">
+                                Dropdown link
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a class="dropdown-item" href="#">Action</a></li>
+                                <li><a class="dropdown-item" href="#">Another action</a></li>
+                                <li><a class="dropdown-item" href="#">Something else here</a></li>
+                            </ul>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </section>
+    <!-- Section For Navbar -->
+
+
+    <!-- Section For Body -->
+    <section class="main-content">
+        <div class="text-center">
+            <h2 class="greetings-teks fw-lighter">Simple Systeme Fordiscount</h2>
+            <h6 class="fw-lighter pt-2 pb-4">Integer Cursus Augue Ac Cursus</h6>
+        </div>
+        <div class="container text-center mt-5">
+            <div class="main-content row">
+                <div class="col-lg-4 col-md-4 col-sm-4">
+                    <div class="pict1">
+                        <img class="img-fluid mb-3" src="./assets/img/cloud.svg" alt="Clude Picture">
+                        <h5>INTRODUCING WHATSAPP</h5>
+                        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit</p>
+                        <a href="#">Read More</a>
+                    </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)

Reply:
Siap makasih informasinya mas